pkgs/top-level/all-packages.nix

```diff
@@ -18118,9 +18118,11 @@ with pkgs;
 
   b4 = callPackage ../development/tools/b4 { };
 
-  babeltrace = callPackage ../development/tools/misc/babeltrace/v1.nix { };
+  babeltracePackages = callPackage ../development/tools/misc/babeltrace { };
```

Comment:
This is too much IMO..

Reply:
This was recommended [here](https://github.com/NixOS/nixpkgs/pull/285681#issuecomment-1922860408). It think that using the recommended `babeltraceVersions` would be better than current `babeltracePackages` so I changed it.

Reply:
> This was recommended [here](https://github.com/NixOS/nixpkgs/pull/285681#issuecomment-1922860408). It think that using the recommended `babeltraceVersions` would be better than current `babeltracePackages` so I changed it.

I liked most this option:

> * Use something like `inherit (callPackages ...) babeltrace babeltrace 2`

I think there is no value in a `babeltracePackages` attribute set, when the attributes within this attribute set don't interact with each other, and there are just 2 of them. Also note it creates a bit more noise when you `nix search`.